src/main/java/milutrock/Parser.java

```diff
@@ -0,0 +1,124 @@
+package milutrock;
+
+import java.util.ArrayList;
+
+import milutrock.exceptions.InvalidTaskFormatException;
+import milutrock.exceptions.UnknownCommandException;
+import milutrock.tasks.Deadline;
+import milutrock.tasks.Event;
+import milutrock.tasks.Task;
+import milutrock.tasks.ToDo;
+
+/**
+ * Parses user input commands to interact with a `TaskList` and `Ui`.
+ */
+public class Parser {
+    private String stdin;
+    private TaskList taskList;
+    private Ui ui;
+
+    public Parser(TaskList taskList, Ui ui) {
+        this.stdin = "";
+        this.taskList = taskList;
+        this.ui = ui;
+    }
+
+    /**
+     * Returns the value of the `stdin` variable.
```

Comment:
Perhaps the comment could be aligned with the one below, where it specifies user input over stdin